cli/common.go

```diff
@@ -62,7 +93,7 @@ func createRequest(dirName string) (controller.Request, error) {
 		req.Units = append(req.Units, controller.Unit{Name: name, Content: content})
 	}
 
-	sliceIDs, err := createSliceIDs(dirName)
+	sliceIDs, err := createSliceIDs(slices)
```

Comment:
My sugar reasoning was against having this stuff in the cli package. Anyway, lets see where we are going.